docs/getting-started/first-source.md

```diff
@@ -1,14 +1,14 @@
-In this tutorial, you will be using the [CloudEvents Player](https://github.com/ruromero/cloudevents-player) to showcase the core concepts of Knative Eventing. By the end of this tutorial, you should have something that looks like this:
+In this tutorial, you will be using the [CloudEvents Player](https://github.com/ruromero/cloudevents-player){target=blank} to showcase the core concepts of Knative Eventing. By the end of this tutorial, you should have something that looks like this:
 
 <figure>
-  <img src="https://user-images.githubusercontent.com/16281246/116408922-589c3d00-a801-11eb-9835-5c37ff57c861.png" draggable="false">
+  <img src="../images/event_diagram.png" draggable="false">
   <figcaption>Figure 6.6 from <a href = "https://www.manning.com/books/knative-in-action" target="_blank">Knative in Action</a> <br>
 Here, the CloudEvents Player is acting as both a <code>Source</code> and a <code>Sink</code> for CloudEvents.
   </figcaption>
 </figure>
 
 ## Creating your first Source
-The CloudEvents Player acts as a `Source` for CloudEvents by intaking the URL of our `Broker` as an environment variable (`BROKER_URL`) and sending CloudEvents via the UI.
+The CloudEvents Player acts as a Source for CloudEvents by intaking the URL of our Broker as an environment variable (`BROKER_URL`) and sending CloudEvents via the UI.
```

Comment:
The CloudEvents Player acts as a Source for CloudEvents by taking the URL of the Broker as an environment variable, `BROKER_URL`, and sending CloudEvents through the UI.

Reply:
Changed this to 

The CloudEvents Player acts as a Source for CloudEvents by intaking the URL of the Broker as an environment variable, `BROKER_URL`. You will send CloudEvents to the Broker through the CloudEvents Player application.